### ISSUE TITLE
Issue with OLE2_OBJECT (Microsoft interface)

### ISSUE THREAD

**Issue by conlutio** [SAP/abap-cleaner#175]
Hi!

I think this is rarely used, but occasionally the OLE2_OBJECT is used to call the interface to Microsoft products.

```
  DATA: lr_outlook        TYPE ole2_object,
        lr_mail           TYPE ole2_object,
        lr_ole_attachment TYPE ole2_object.
...

  " Create handler for outlook
  CREATE OBJECT lr_outlook 'Outlook.Application' NO FLUSH.

  " Create new mail
  CALL METHOD OF
      lr_outlook
      'CreateItem' = lr_mail
    EXPORTING
      #1           = 0.
```
The ABAP cleaner converts these lines as follows:
```
" Create handler for outlook
  lr_outlook = NEW #( 'Outlook.Application' NO FLUSH ).

  " Create new mail
  OF(
      lr_outlook
      'CreateItem' = lr_mail
      EXPORTING #1 = 0 ).
```
This, of course, gives us a syntax error. The rules that intervene here are:

[Replace CREATE OBJECT with NEW constructor](https://github.com/SAP/abap-cleaner/blob/main/docs/rules/CreateObjectRule.md#replace-create-object-with-new-constructor)

[Replace CALL METHOD with functional call](https://github.com/SAP/abap-cleaner/blob/main/docs/rules/CallMethodRule.md#replace-call-method-with-functional-call)

Perhaps it is possible to recognise that the data type is OLE2_OBJECT, and in that case exclude the coding from the rule?

Kind Regards,
Jürgen

**Comment by jmgrassau** [comment]
Hi Jürgen, 

thanks a lot for reporting these bugs! These cases were indeed not yet considered, but the issue will be fixed with the next release.

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Jürgen, 

thanks again for reporting this! The issues should now be fixed in version 1.9.0, which was just released.

Kind regards,
Jörg-Michael
